### PR TITLE
docs: python upload guide

### PR DESCRIPTION
**Pull request by jrasm91**

### CODE REVIEW

**Review by danieldietzler — APPROVED** [review]
Nice!

File: docs/docs/guides/python-file-upload.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    stats = os.stat(file)
+
+    headers = {
+        'Accept': 'application/json',
+        'x-api-key': API_KEY
+    }
+
+    data = {
+        'deviceAssetId': f'{file}-{stats.st_mtime}',
+        'deviceId': 'python',
```
Comment (bo0tzz) [inline docs/docs/guides/python-file-upload.md:24]: Do we still require these?

Comment (jrasm91) [reply docs/docs/guides/python-file-upload.md:24]: oh yeah

Comment (jrasm91) [reply docs/docs/guides/python-file-upload.md:24]: they aren't even nullable in the database

Comment (jrasm91) [reply docs/docs/guides/python-file-upload.md:24]: gotta tell those mobile developers to stop using them already

**Pull request merged**